src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpAuthHelper.cs

```diff
@@ -302,6 +302,10 @@ public void SaveServerCredentialsToCache(Uri uri, uint authScheme, ICredentials
             uint authScheme,
             uint authTarget)
         {
+            string userName;
+            string password;
+            string domain;
```

Comment:
Does ```domain``` need to be declared here?  It seems like it's only used inside the below else block.